rep-2000.rst

```diff
@@ -890,45 +890,45 @@ Minimum language requirements:
-+-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
-| Cyclone DDS |                                          0.9.x (Papillons)                                                           |
-+-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
-| Fast-DDS    |                                          2.6.x                                                                       |
-+-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
-| Connext DDS |              6.0.1              |   N/A    |              6.0.1             |                  N/A                   |
-+-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
-| Gurum DDS   |              2.7.x              |   N/A    |     2.7.x      |              N/A                                       |
-+-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
++----------------+---------------------------------+------------------------------------------------------------------------------------+
+|                |        Required Support         |                        Recommended Support                                         |
++----------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Package        |  Ubuntu Jammy  |  Windows 10**  |  RHEL 8  |  Ubuntu Focal  |    macOS**    | Debian Bullseye  |   OpenEmbedded**    |
++================+================+================+==========+================+===============+==================+=====================+
+| CMake          |     3.22.1     |     3.22.0     |  3.20.2  |     3.16.3     |    3.14.4     |      3.18.4      | 3.22.3 / 3.16.5***  |
++----------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| EmPY           |      3.3.4     |                                                3.3.2                                                |
++----------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Gazebo Classic |    11.x.x*     |       N/A      |   N/A    |    11.0.0*     |    11.x.x     |      11.x.x*     |         N/A         |
++----------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Gazebo (Ign)   |    Fortress*   |       N/A      |   N/A    |    Fortress*   |   Fortress*   |     Fortress*    |         N/A         |
```

Comment:
I'm wondering if we should change this to explicitly call out "Ignition" (rather than just "Ign"), to be fully clear.  @azeey what do you think?

Reply:
Yes, "Ignition" makes sense

Reply:
OK, I switched it to "Ignition" now.  It still seems to render OK, so we'll go with that.  I'll go ahead and approve and merge this.